apisix/consumer.lua

```diff
@@ -101,46 +152,21 @@ local function plugin_consumer()
                 if not plugins[name] then
                     plugins[name] = {
                         nodes = {},
+                        len = 0,
```

Comment:
The purpose of introducing the len variable here is to reduce the redundant computation of the table length. Refer to the `table.insert` section in this link:  https://api7.ai/learning-center/openresty/lua-table-and-metatable